include/nleobs.h

```diff
@@ -18,16 +21,18 @@ typedef struct nle_observation {
     unsigned char *chars;    /* Size ROWNO * (COLNO - 1) */
     unsigned char *colors;   /* Size ROWNO * (COLNO - 1) */
     unsigned char *specials; /* Size ROWNO * (COLNO - 1) */
-    long *blstats;           /* NLE_BLSTATS_SIZE */
+    long *blstats;           /* Size NLE_BLSTATS_SIZE */
     unsigned char *message;  /* Size NLE_MESSAGE_SIZE */
-    int *program_state;      /* NLE_PROGRAM_STATE_SIZE */
-    int *internal;           /* NLE_INTERNAL_SIZE */
-    short *inv_glyphs;       /* NLE_INVENTORY_SIZE */
+    int *program_state;      /* Size NLE_PROGRAM_STATE_SIZE */
+    int *internal;           /* Size NLE_INTERNAL_SIZE */
+    short *inv_glyphs;       /* Size NLE_INVENTORY_SIZE */
     unsigned char
-        *inv_strs; /* NLE_INVENTORY_SIZE * NLE_INVENTORY_STR_LENGTH */
-    unsigned char *inv_letters;  /* NLE_INVENTORY_SIZE */
-    unsigned char *inv_oclasses; /* NLE_INVENTORY_SIZE */
+        *inv_strs; /* Size NLE_INVENTORY_SIZE * NLE_INVENTORY_STR_LENGTH */
+    unsigned char *inv_letters;  /* Size NLE_INVENTORY_SIZE */
+    unsigned char *inv_oclasses; /* Size NLE_INVENTORY_SIZE */
     unsigned char *screen_descriptions;  /* Size ROWNO * (COLNO - 1) * NLE_SCREEN_DESCRIPTION_LENGTH */
+    unsigned char *terminal_chars;       /* Size NLE_TERM_LI * NLE_TERM_CO */
```

Comment:
I know I keep arguing about observation names but what do you think about "tty_chars"?